src/Pages/CreateRecruitment/page.tsx

```diff
@@ -0,0 +1,301 @@
+                        defaultValue={parseSelectValue('applicantCount')}
+                        values={APPLICATION_CNT}
+                        {...field}
+                      />
+                    )}
+                  />
+                </LabelForm>
+                <LabelForm name="applicantCount" label="모집 마감일" errors={errors}>
+                  <CalendarButton>
+                    <Controller
+                      control={control}
+                      name="recruitmentEndDateTime"
+                      rules={{ required: CREATE_RECRUITMENT.recruitmentEndDateTime }}
+                      render={({ field }) => (
+                        <EndDate {...field} defaultValue={parseSelectValue('recruitmentEndDateTime') as string} />
+                      )}
+                    />
+                  </CalendarButton>
+                </LabelForm>
+                <LabelForm name="positionIds" errors={errors}>
+                  <Controller
+                    control={control}
+                    name="positionIds"
+                    rules={{ required: CREATE_RECRUITMENT.positionIds }}
+                    render={({ field }) => (
+                      <CustomSelect
+                        label="포지션"
+                        placeholder="포지션"
+                        defaultValue={parseSelectValue('positionIds')}
+                        values={POSITION}
+                        isMulti
+                        {...field}
+                      />
+                    )}
+                  />
+                </LabelForm>
+                <LabelForm label="기술 스택">
+                  <InputText
+                    onClick={() => setIsOpen(!isOpen)}
+                    placeholder={DEF_STACK_PLACEHOLDER}
+                    value={content === DEF_STACK_PLACEHOLDER ? null : content}
+                  />
+                  {isOpen && (
+                    <StackModal
+                      handleModal={setIsOpen}
+                      initialSelectedStacks={selectedStacks ?? []}
+                      handleSelectedStacks={handleSelectedStacks}
+                    />
+                  )}
+                  {selectedStacks?.length === 0 && <ErrorMsg>{'스택을 선택해주세요'}</ErrorMsg>}
+                </LabelForm>
+                <LabelForm errors={errors} name="contact">
+                  <Controller
+                    control={control}
+                    name="contact"
+                    rules={{ required: CREATE_RECRUITMENT.contact }}
+                    render={({ field }) => (
+                      <CustomSelect
+                        label="연락방법"
+                        placeholder="연락방법"
+                        defaultValue={parseSelectValue('contact')}
+                        values={CONTACT}
+                        {...field}
+                      />
+                    )}
+                  />
+                </LabelForm>
+                <LabelForm label="연결 url" errors={errors}>
```

Comment:
```suggestion
                <LabelForm label="연결 url" name="callUrl" errors={errors}>
```

name이 빠져있어 연결 URL의 validation이 올바르게 동작하지 않습니다.

아래 사진은 피그마 디자인 시안입니다.

![image](https://github.com/Ludo-SMP/ludo-frontend/assets/72962900/2fa7425d-e912-4789-a17b-3c2423d05cb0)

아래 사진은 로컬에서의 테스트 결과입니다.

![image](https://github.com/Ludo-SMP/ludo-frontend/assets/72962900/97cd2b42-0cfe-465a-b97c-9b5e1d7eaa18)

Reply:
아 지금 보니까 기술 스택 부분도 validation이 빠진 것 같아요!

![image](https://github.com/Ludo-SMP/ludo-frontend/assets/72962900/ce076722-32e9-4d76-af59-3ef08de800df)

Reply:
앗 callUrl name을 등록하는 걸 놓쳤네요. 반영하겠습니다 확인 감사해요!

`props`로 `error`랑 `name`은 같이 전달이되어야 하는데요, 혹시 둘중 하나가 누락된 경우 타입으로 체크하는 방법이 있을까요? 🤔

Reply:
글쎄요... 일단 이런 방법이 있지 않을까요?

```ts
{ a: T, b: T } | { ... }
```

이렇게 하면 `a`또는 `b` 둘다 있거나 둘다 아예 없거나 하는 경우만 통과되는 것으로 알긴 하지만, 좀더 같이 더 나은 방법을 테스트해보면 좋을 것 같아요!

예를 들면 error를 처음부터 받아서 factory pattern 형태로 만든다던가... 그럼 항상 error가 항상 들어가 있겠죠?
아니면 `useContext`로 에러를 주는 방법도 있는데 그건 너무 복잡할 거 같고... 저도 명확한 해결책은 떠오르지가 않네요.

Reply:
기술 스택 validation은 이슈가 있어서 조금 찾아보고 있었습니다.
기존에는 `onSubmit`을 호출한 시점에 기술스택 선택이 안된 경우, 에러메시지를 띄우는 순서로 작업을 했습니다.

그런데, `RHF`는 `onSubmit`을 호출하기 전에 유효성 검사를 하기 때문에, 유효하지 않은 값이 있는 경우 
`onSubmit`이 실행되지 않는 문제가 있었어요. 
(rhf에서 관리하는 값들이 다 validate된 후에, onSubmit 호출 가능 -> 기술스택 validation 로직 실행)

Reply:
찾아보니 `handleSubmit` 두번째 인자로 `onError`를 넘길 수 있어서 해당 함수에 기술스택 validataion 로직 추가했습니다 
확인 감사해요! 🙂

참고: https://github.com/orgs/react-hook-form/discussions/8372